base/v0_5_exp/translate.go

```diff
@@ -217,6 +218,48 @@ func translateLink(from Link, options common.TranslateOptions) (to types.Link, t
 	return
 }
 
+func translateSSHAuthorizedKeysLocal(from []SSHAuthorizedKeyFile, options common.TranslateOptions) (to []types.SSHAuthorizedKey, tm translate.TranslationSet, r report.Report) {
+	tr := translate.NewTranslator("yaml", "json", options)
+	tm, r = translate.Prefixed(tr, "ssh_authorized_keys_local", &from, &to)
```

Comment:
This shouldn't be here, since you're manually handling both the field conversions and the `TranslationSet` below.